app/src/main/java/com/neiapp/spocan/Models/User.java

```diff
@@ -35,18 +56,23 @@ public String toJson() {
 
         json.addProperty("nickname", this.nickname);
         json.addProperty("type", this.type.getId());
+        json.addProperty("amountOfInitiatives",this.amountOfInititatives);
+        json.addProperty("admin", this.admin);
 
         return json.toString();
     }
 
-    public static User convertJson(String jsonToTransform){
+    public static User convertJson(String jsonToTransform) {
         try {
             JSONObject jsonObject = new JSONObject(jsonToTransform);
             final String nickname = jsonObject.getString("nickname");
             final int typeId = jsonObject.getInt("type");
-            return new User(nickname, UserType.fromIdOrElseThrow(typeId));
+            final boolean admin = jsonObject.getBoolean("admin");
+            final int amountOfInitiatives = jsonObject.getInt("amountOfInitiatives");
+            return new User(nickname, UserType.fromIdOrElseThrow(typeId),amountOfInitiatives, admin);
```

Comment:
El atributo que definiste en el BE se llama 'amount_of_initiatives', osea con Snake Case (guión bajo para separar espacios). Debería estar la misma convención aquí.